src/components/navigation/header/config/showUnderMoreConstants.ts

```diff
@@ -0,0 +1,14 @@
+import { breakpoints } from 'src/themes';
+
+export type ShowUnderMoreBreakpoint = `(max-width: ${number}px)`;
+
+const minWidthMore = 500;
+const maxWidthMore = breakpoints.md.px;
+
+export const sell: ShowUnderMoreBreakpoint = `(max-width: ${breakpoints.sm.px}px)`;
```

Comment:
Can you elaborate on the concept here? (I struggle to follow the configuration now. Before we had  `showUnderMoreLinkBelow: md` indicating the visibility of the configured element alternates on the specified breakpoint. now with `showUnderMoreLinkBelow: estimate` I have no indication at all without digging into the code)

Reply:
before everything was duplicated in 2 places and there were already mistakes caused by this. Now, everything is in 3 places. That is why I extracted it.

Reply:
I meant more on naming the breakpoints after navigation points instead of having speaking names that let one know when the change happens based on the config (Like why can't it just be the same prop in the config you move to separate files?)

Reply:
done ✅